diff --git a/meson.build b/meson.build
index 1833440..0f95da4 100644
--- a/meson.build
+++ b/meson.build
@@ -26,7 +26,7 @@ add_global_link_arguments(cpp_link_args, language : 'cpp')
 
 thread_dep = dependency('threads')
 libinput = dependency('libinput')
-fmt = dependency('fmt', fallback: ['fmtlib', 'fmt_dep'])
+fmt = subproject('fmtlib').get_variable('fmt_dep')
 wayland_client = dependency('wayland-client')
 wayland_cursor = dependency('wayland-cursor')
 wayland_protos = dependency('wayland-protocols')
