diff --git a/meson.build b/meson.build
index 1833440..b8ff924 100644
--- a/meson.build
+++ b/meson.build
@@ -97,7 +97,7 @@ executable(
 install_data(
     './resources/config',
     './resources/style.css',
-    install_dir: '/etc/xdg/waybar',
+    install_dir: join_paths(get_option('outprefix'), 'etc/xdg/waybar'),
 )
 
 clangtidy = find_program('clang-tidy', required: false)
diff --git a/meson_options.txt b/meson_options.txt
new file mode 100644
index 0000000..512f77c
--- /dev/null
+++ b/meson_options.txt
@@ -0,0 +1,2 @@
+option('outprefix', type: string, value : '/', description: 'output prefix directory')
+
