From 8bd19591b9a415ad8bdaef42684602a91e06cc97 Mon Sep 17 00:00:00 2001
From: Cole Mickens <cole.mickens@gmail.com>
Date: Sun, 21 Oct 2018 23:08:16 -0700
Subject: [PATCH] meson: add libdrm dependency

---
 meson.build     | 1 +
 src/meson.build | 1 +
 2 files changed, 2 insertions(+)

diff --git a/meson.build b/meson.build
index 6ea40b0..5491a26 100644
--- a/meson.build
+++ b/meson.build
@@ -14,6 +14,7 @@ wayland_client    = dependency('wayland-client')
 wayland_protocols = dependency('wayland-protocols', version: '>=1.14')
 
 #Library deps
+libdrm        = dependency('libdrm')
 libavutil     = dependency('libavutil')
 libavcodec    = dependency('libavcodec')
 libavformat   = dependency('libavformat')
diff --git a/src/meson.build b/src/meson.build
index e943fe5..a59a838 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -4,6 +4,7 @@ executable(
     install: true,
     dependencies: [wayland_client,
                    wlstream_protos,
+		   libdrm,
                    libavutil,
                    libavcodec,
                    libpulse,
-- 
2.16.2

